host/CMakeLists.txt

```diff
@@ -469,15 +469,37 @@ if (CMAKE_C_COMPILER_ID MATCHES GNU)
 endif ()
 
 # Use the same the compile options and definitions from oehost.
-target_compile_options(
-  oehostverify
-  PRIVATE $<TARGET_PROPERTY:oehost,COMPILE_OPTIONS>
-  INTERFACE $<TARGET_PROPERTY:oehost,INTERFACE_COMPILE_OPTIONS>)
-
 target_compile_definitions(
```

Comment:
One more comment: Instead of doing this (here and the other newly added code), can we just set those compile definitions/options at the same places as `oehost`? See what we did in line 345&347, 349&350 (it would be good to flip the order: oehost then oehostverify, making it consistent), 413&415 for examples.